ports/pipewire/portfile.cmake

```
@@ -0,0 +1,83 @@
+vcpkg_from_gitlab(
+    GITLAB_URL https://gitlab.freedesktop.org/
+    OUT_SOURCE_PATH SOURCE_PATH
+    REPO pipewire/pipewire
+    REF 0.3.52
+    SHA512 30e9cf74c92babafe386f02a03bb5c41a8ee5591a02f15845cca1ee44f091ce68eb14d48943d43b680cb525026a19e0290997670f9a82156eaa72e974fe6d01a
+    HEAD_REF master # branch name
+)
+
+vcpkg_configure_meson(
+    SOURCE_PATH ${SOURCE_PATH}
```

```suggestion
    SOURCE_PATH "${SOURCE_PATH}"
```
Please also add double qoutes to other absolute paths.